jetty-webapp/src/test/java/org/eclipse/jetty/webapp/WebAppContextTest.java

```diff
@@ -254,177 +266,184 @@ public void testNullPath() throws Exception
         server.addConnector(connector);
 
         server.start();
-        try
-        {
-            String response = connector.getResponse("GET http://localhost:8080 HTTP/1.1\r\nHost: localhost:8080\r\nConnection: close\r\n\r\n");
-            assertThat("Response OK", response, containsString("200 OK"));
-        }
-        finally
-        {
-            server.stop();
-        }
+
+        String rawResponse = connector.getResponse("GET http://localhost:8080 HTTP/1.1\r\nHost: localhost:8080\r\nConnection: close\r\n\r\n");
+        HttpTester.Response response = HttpTester.parseResponse(rawResponse);
+        assertThat("Response OK", response.getStatus(), is(HttpStatus.OK_200));
```

Comment:
Where's server.stop()?

Reply:
In `tearDown()`, we had many cases with servers that were started, but not stopped.
Not it's always done.

Reply:
See `tearDown()` as a result of tracking the `newServer()` created instance.
Not all of the Server instances created in the test were being stopped.
Now they are.